app/forms/hyrax/etd_form.rb

```diff
@@ -4,8 +4,7 @@ class EtdForm < Hyrax::Forms::WorkForm
     SINGLE_VALUE = [:degree, :school, :department].freeze
 
     self.model_class = ::Etd
-    self.terms += [:degree, :date, :date_label, :department, :institution,
-                   :orcid_id, :resource_type, :rights_note, :school]
+    self.terms += [:degree, :date, :date_label, :department, :institution, :orcid_id, :resource_type, :rights_note, :school]
```

Comment:
Fixes Rubocop array alignment error.

Reply:
@jenlindner where is this error coming from? If this file hasn't changed otherwise, CI has been running rubocop all along, and we shouldn't need to do style-only fixups.

Are you running the correct rubocop configs in your local environment? You may need to do `bundle exec rubocop` instead of just `rubocop` if your system versions of `bixby`, `rubocop`, `rubocop-rspec`, and/or `rubocop-rails` are different from those in the application's bundle.

Reply:
I have `be` aliased to `bundle exec` and use it with individual specs and rake. `be rake` gives me a warning from this version of bixby `ruby/2.4.2/gems/bixby-0.3.1/bixby_default.yml` and bundle show rubocop gives me `ruby/2.4.2/gems/rubocop-0.50.0`